lib/box/selinux.go

```diff
@@ -33,19 +34,25 @@ import (
 // if the domain cannot be determined.
 // Assumes SELinux support is on
 func getSELinuxProcLabel(rootfs, cmd string) (label string) {
+	logger := log.WithField("path", cmd)
```

Comment:
Nit: Should this field maybe be added to the logger after the "abspath" has been determined? Just so it contains the actual command path.